src/components/NcActions/NcActions.vue

```diff
@@ -1091,7 +1218,7 @@ export default {
 		 * @return {boolean}
 		 */
 		isValidSingleAction(action) {
-			return ['NcActionButton', 'NcActionLink', 'NcActionRouter'].includes(this.getActionName(action))
+			return ['NcActionButton', 'NcActionLink', 'NcActionRouter'].some(valid => this.getActionName(action).startsWith(valid))
```

Comment:
Why change to `startsWith`?

Reply:
I thought we could introduce a naming convention for these wrappers. So if the components name starts with NcActions, we just pass it on. But that wasn't fully thought through and didn't work yet.

Reply:
As an alternative, we could ask to add a custom option and check `$options`.

But this won't help in a general case. Developers may make wrappers for a list of actions

```html
<template>
  <div>
    <NcActionButton />
    <NcActionInput />
  </div>
</template>
```

Or dynamic actions. Like, I dunno,
```html
<template>
  <NcActionButton v-if="iLikeButtons" /> 
  <NcActionInput v-else-if="iDont" />
</template>
```

Reply:
> Or dynamic actions. Like, I dunno

Have this in forms: https://github.com/nextcloud/forms/blob/e1cfbc737234e0bd3ff10f8a1222223a8093e254/src/views/Results.vue#L134

Reply:
> Have this in forms:

This will work. I meant when those `<template>...</template>` are templates of new custom components.

```html
<template>
  <div>
    <NcActionButton />
    <NcActionInput />
  </div>
</template>

<script>
export default {
  name: 'MyAction'
}
</script>  
```

and then

```html
<template>
  <NcActions>
    <MyAction />
  </div>
</template>
```